src/PaloAltoNetworks/PaloAltoNetworks.Autorest/test/Get-AzPaloAltoNetworksFirewall.Tests.ps1

```diff
@@ -0,0 +1,33 @@
+if(($null -eq $TestName) -or ($TestName -contains 'Get-AzPaloAltoNetworksFirewall'))
+{
+  $loadEnvPath = Join-Path $PSScriptRoot 'loadEnv.ps1'
+  if (-Not (Test-Path -Path $loadEnvPath)) {
+      $loadEnvPath = Join-Path $PSScriptRoot '..\loadEnv.ps1'
+  }
+  . ($loadEnvPath)
+  $TestRecordingFile = Join-Path $PSScriptRoot 'Get-AzPaloAltoNetworksFirewall.Recording.json'
+  $currentPath = $PSScriptRoot
+  while(-not $mockingPath) {
+      $mockingPath = Get-ChildItem -Path $currentPath -Recurse -Include 'HttpPipelineMocking.ps1' -File
+      $currentPath = Split-Path -Path $currentPath -Parent
+  }
+  . ($mockingPath | Select-Object -First 1).FullName
+}
+
+Describe 'Get-AzPaloAltoNetworksFirewall' {
+    It 'List' -skip {
```

Comment:
Please add tests for those new added cmdlets.

Reply:
There are no newly added cmdlets, correct me please if you see any.

Reply:
@dolauli I figured out that these tests are not skipped but are duplicates. All cmdlets scenarios are already covered in **AzPaloAltoNetworks.Tests.ps1** in create, get, update, remove fashion. I have updated this PR 
![image](https://github.com/Azure/azure-powershell/assets/152487298/4b405d65-e466-4c66-bc2d-9507d7e1644c)